thirtyinch/src/main/java/net/grandcentrix/thirtyinch/serialize/TiPresenterSerializer.java

```diff
@@ -0,0 +1,40 @@
+package net.grandcentrix.thirtyinch.serialize;
+
+import android.support.annotation.NonNull;
+
+import net.grandcentrix.thirtyinch.TiPresenter;
+import net.grandcentrix.thirtyinch.TiView;
+
+/**
+ * Interface to serialize and deserialize presenters. The interface should be implemented, if
+ * a {@link TiPresenter} should be restored, after a process has died, but the UI component is
+ * being recreated.
+ */
+public interface TiPresenterSerializer {
```

Comment:
@passsy After reviewing all the changes in the 0.8.0 version I think this interface can be removed and the new `TiPresenterSavior` could be used instead. After that it would be possible for me to release the module in #51 separately and I could finally stop working on the fork.

The only changes that would be necessary to make this work is to add the `TiPresenterSavior` as argument in the `TiConfiguration`. This way I get a chance to replace the default savior. 

The 2nd change that's necessary is to modify the recover method in the savior interface `TiPresenter recover(String presenterId, @NonNull Object host);`. Here a third parameter would be necessary that you've already implemented: `TiPresenterProvider`. In most cases that would be the `TiFragment` or `TiActivity`. That would give me the option to construct a new presenter in case it can be recovered after the process died.

Would that be possible?